CHANGES

```diff
@@ -3,6 +3,38 @@ repositories (changes that are automatically handled by the format upgrade tools
 are not marked). Those prefixed with "(+)" are new command/option (since
 2.1.0~alpha2).
 
+2.2.0~beta1:
+* (*) Check and advertise to use Git for Windows [#5718 @rjbou - fix #5617]
+  * (+) Add the `--git-location` and `--no-git-location` arguments
+  * (+) Add a new `git-location` option on Windows
```

Comment:
```suggestion
* Check and advertise to use Git for Windows [#5718 @rjbou - fix #5617]
  * (+) Add the `--git-location` and `--no-git-location` options for Windows
```

Reply:
The second part talks about the option (as-in `opam option`)

Reply:
oh, i see!

Reply:
then, i propose
```suggestion
* Check and advertise to use Git for Windows [#5718 @rjbou - fix #5617]
  * (+) Add the `--git-location` and `--no-git-location` cli arguments
  * (+) Add a new `git-location` modifiable opam option on Windows
```